.gitignore

```diff
@@ -3,15 +3,3 @@
 # If you find yourself ignoring temporary files generated by your text editor
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
-
-# Ignore bundler config.
-/.bundle
```

Comment:
I think you'd always want to add .cache files to your gitignore since you have a lot of these extraneous files that were included in your PR